app/views/version-5/layouts/main.html

```diff
@@ -63,6 +63,18 @@
      
   </div>
 </header>
+<div class="fiat-phase-banner">
+<div class="govuk-phase-banner govuk-width-container">
+  <p class="govuk-phase-banner__content">
+    <strong class="govuk-tag govuk-phase-banner__content__tag">
+      Beta
+    </strong>
+    <span class="govuk-phase-banner__text">
+      This is a new service – your <a class="govuk-link" href="https://mcas-proxyweb.mcas.ms/certificate-checker?login=false&originalUrl=https%3A%2F%2Fforms.office.com.mcas.ms%2Fe%2F333tRg06Sq%3FMcasTsid%3D26110&McasCSRF=2064d3faa6bd6d60ca6af79b9fcf55d141558344814b0ff0f58b42ab1a68d277">feedback (opens in a new tab)</a> will help us to improve it.
```

Comment:
```suggestion
      This is a new service – your <a class="govuk-link" target="blank" href="https://mcas-proxyweb.mcas.ms/certificate-checker?login=false&originalUrl=https%3A%2F%2Fforms.office.com.mcas.ms%2Fe%2F333tRg06Sq%3FMcasTsid%3D26110&McasCSRF=2064d3faa6bd6d60ca6af79b9fcf55d141558344814b0ff0f58b42ab1a68d277">feedback (opens in a new tab)</a> will help us to improve it.
```
This link doesn't actually open in a new page - add target="blank" before 'href' to make it open in a new tab - [here is an example](https://www.w3schools.com/tags/att_a_target.asp)